test/clj/game_test/cards/resources.clj

```diff
@@ -803,6 +803,22 @@
       (prompt-choice :runner "Card from hand")
       (prompt-choice :runner "Steal"))))
 
+(deftest gang-sign-correct-prompts
+         ;; Gang Sign - accessing from HQ, not including root. Issue #2113.
+   (do-game
+     (new-game (default-corp [(qty "Hostile Takeover" 1) (qty "Snare!" 1)])
+               (default-runner [(qty "Gang Sign" 1)]))
+     (take-credits state :corp)
+     (play-from-hand state :runner "Gang Sign")
+     (take-credits state :runner)
+     (play-from-hand state :corp "Hostile Takeover" "New remote")
+     (score-agenda state :corp (get-content state :remote1 0))
```

Comment:
There is a snazzy `play-and-score` function that condenses these two functions into one.